tests/check_ipc.c

```diff
@@ -265,10 +265,12 @@ s1_msg_process_fn(qb_ipcs_connection_t *c,
 		}
 
 	} else if (req_pt->id == IPC_MSG_REQ_SERVER_FAIL) {
+		qb_ipcs_destroy(s1);
 		exit(0);
 	} else if (req_pt->id == IPC_MSG_REQ_SERVER_DISCONNECT) {
 		multiple_connections = QB_FALSE;
 		qb_ipcs_disconnect(c);
+		qb_ipcs_destroy(s1);
```

Comment:
Thanks. testing this on FreeBSD causes breakage so I think the qb_ipcs_destroy() calls cause the sockets to be closed when the client test is expecting them to be still there. I think I might have to revert to the manual unlink that I had to use into test_ipcc_truncate_when_unlink_fails_shm()